resources/chefdk/msi/source.wxs.erb

```diff
@@ -65,13 +65,26 @@
           </Directory>
         </Directory>
       </Directory>
-      
+
       <Directory Id="ProgramMenuFolder">
         <Component Id="StartMenuShortcut" Guid="{6DD3FFF3-E009-40E7-B1C2-EF606B941CCD}">
-            <Shortcut Id="StartMenuShortcutDef" 
-                      Name="!(loc.ChefDkStartMenuShortcutDefName)"
-                      Description="!(loc.ChefDkStartMenuShortcutDefDescription)"
-                      Target="[#StartChefDkScript]"/>
```

Comment:
Need to confirm what the # is actually doing.. I see examples without the # in the WIX books.